### PR TITLE
[MOSIP-29616]Updated mosip-file-server for release-1.2.0.1

### PR DESCRIPTION
**Pull request by abhishek8shankar**

### CODE REVIEW

File: deployment/v3/mosip/mosip-file-server/values.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosip-file-server/values.yaml:15]: please remove values.yaml file as it is not required here

File: deployment/v3/mosip/mosip-file-server/README.md [hunk 2]
```diff
@@ -11,4 +11,4 @@ The chart here installs a mosip-file-server accessed over an https URL.
 ./install.sh
 ```
 ## Download
-The download URL will be available at `https://your-fileserver-host`. Example: `https://fileserver.sandbox.xyz.net`.
+The download URL will be available at `https://your-fileserver-host`. Example: `https://fileserver.sandbox.xyz.net`.
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosip-file-server/README.md:14]: please change this point to the below sentence

The download URL will be available at `https://api-host`. Example: `https://api.sandbox.xyz.net`.

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
README.md has to be updated.

File: deployment/v3/mosip/mosip-file-server/values.yaml (hunk 1 above)
Comment (abhishek8shankar) [reply deployment/v3/mosip/mosip-file-server/values.yaml]: removed the file.

File: deployment/v3/mosip/mosip-file-server/install.sh [hunk 3]
```diff
@@ -6,8 +6,9 @@ if [ $# -ge 1 ] ; then
   export KUBECONFIG=$1
 fi
 
+
 NS=mosip-file-server
-CHART_VERSION=12.0.1
+CHART_VERSION=12.0.2
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply deployment/v3/mosip/mosip-file-server/install.sh]: reverted to 12.0.1

File: deployment/v3/mosip/mosip-file-server/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# Regclient 
+# Regclient
```
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply deployment/v3/mosip/mosip-file-server/README.md]: Renamed to mosip-file-server

File: deployment/v3/mosip/mosip-file-server/README.md (hunk 2 above)
Comment (abhishek8shankar) [reply deployment/v3/mosip/mosip-file-server/README.md]: edited as required above.

File: deployment/v3/mosip/mosip-file-server/README.md [hunk 5]
```diff
@@ -11,4 +11,4 @@ The chart here installs a mosip-file-server accessed over an https URL.
 ./install.sh
 ```
 ## Download
-The download URL will be available at `https://your-fileserver-host`. Example: `https://fileserver.sandbox.xyz.net`.
+The download URL will be available at https://api-host. Example: https://api.sandbox.xyz.net.
```
Comment (ckm007) [inline deployment/v3/mosip/mosip-file-server/README.md:14]: add all the downloadable links by file server here.

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — APPROVED** [review]

**Review by SwatiPeriwal — APPROVED** [review]
Approved

**Review by ckm007 — APPROVED** [review]

**Pull request merged**